### ISSUE TITLE
Unexpected order of runtimeArgs for execution in launch.json

### ISSUE THREAD

**Issue by SilentDepth** [iradul/vscode-phantomjs-debug#6]
Let's say a `launch.json` written like this:

```
{
  ...
  "file": "path/to/script.js",
  "runtimeArgs": ["ARG1", "ARG2"],
  ...
}
```

Then start debug, and the debugger actually executes something like this:

```
path/to/phantomjs.exe ARG1 ARG2 path/to/script.js
```

However I expected the args order to be `path/to/script.js ARG1 ARG2`. Do I misunderstand something?

phantomjs version: 2.1.1
debugger extension version: 0.0.2

**Comment by iradul** [comment]
Unfortunately it's not possible to pass arguments to the script.
With `runtimeArgs` you can define options that will be passed to phantomjs when it gets executed, options like `--load-images=false` or `--ignore-ssl-errors=true` etc. This is probably more important since you can always workaround passing arguments to your script by hard coding them inside your script.
I'll probably add something like `scriptArgs` in next release.

**Comment by SilentDepth** [comment]
Seems I did misunderstand what `runtimeArgs` is for. I was just trying to avoid any hard code. Thanks for explaination :)

**Comment by iradul** [comment]
I just published new version `0.1.0`. If you update you should be able to use `scriptArgs` array to pass arguments to the script.
